backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/tourplanning/api/impl/OrderDeliveryDayBL.java

```diff
@@ -119,7 +119,7 @@ else if (isUseFallback)
 			int offset = sysConfigBL.getIntValue(
 					SYSCONFIG_Fallback_PreparationDate_Offset,
 					0);
-			order.setPreparationDate(TimeUtil.addDays(order.getDatePromised(), offset));
+			order.setPreparationDate(TimeUtil.addHours(order.getDatePromised(), offset * 24));
```

Comment:
suggestion to avoid these cases:
* pls rename sysconfig constant to SYSCONFIG_Fallback_PreparationDate_Offset_Hours
* pls rename the sysconfig name and append _Hours
* pls make sure in sysconfig's description it's clear we talk about hours